### DISCUSSION TITLE
pkl-intellij 0.30.1 is released

### DISCUSSION THREAD

**Discussion by bioball** [apple/pkl#676] [Announcements]
[pkl-intellij](https://pkl-lang.org/intellij/current/index.html) 0.30.1 is released!

This brings a bugfix and improvements around type inference.

See the [changelog](https://pkl-lang.org/intellij/current/changelog.html#release-0.30.1) for a full list of updates.

**Comment by jamesward** [comment]
Nice! I've noticed that IntelliJ doesn't seem to pickup the updates from the custom plugin repo. I have to uninstall and reinstall. Can we get the plugin in the JetBrains repo instead to avoid this? Or is there something else you can do so we get the newer plugin

**Reply by bioball** [reply]
I've noticed that too. Not really sure what's going on, but yes, we're (still) working on getting the plugin into the JetBrains marketplace. Once that's done, this should all be much easier.

**Reply by odenix** [reply]
@jamesward @bioball JetBrains is telling me that custom plugin repo updates were fixed in IntelliJ 2023.3, and that even before then, restarting IntelliJ would fix the issue. Have you experienced this issue in 2024.x?

**Reply by jamesward** [reply]
I've definitely been experiencing this in 2024.x and will take a screenshot next time I see it.
